### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by MaxSchmide**
- [DEMO LINK](https://maxschmide.github.io/layout_product-cards/)
- [TEST REPORT LINK](https://maxschmide.github.io/layout_product-cards/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -8,6 +8,35 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <img
+        src="./images/imac.jpeg"
+        alt="card img"
```
Comment (And678) [inline src/index.html:14]: Check the [checklist №13](https://github.com/mate-academy/layout_product-cards/blob/master/checklist.md), alt needs to contain information of the image content.

File: src/style.css [hunk 2]
```diff
@@ -1,3 +1,124 @@
...
+  background: url("./images/star.svg") center no-repeat;
+}
+
+.stars__star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.stars--4 .stars__star:nth-child(-n+4) {
+  background: url("./images/star-active.svg") center no-repeat;
+}
```
Comment (And678) [inline src/style.css:81]: Check the description of this task:
```
If you change stars--4 modifier to stars--2 or other - number of selected stars should change correctly.
```
Currently it only works for 4 stars.

File: src/style.css [hunk 3]
```diff
@@ -1,3 +1,124 @@
-body {
+
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+:root {
+  --primary: #060b35;
+  --secondary: #616070;
+}
```
Comment (And678) [inline src/style.css:7]: Nice usage of variables :fire:

File: src/index.html [hunk 4]
```diff
@@ -8,6 +8,35 @@
...
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+        <div class="card__reviews">Reviews: 5</div>
+      </div>
+      <div class="card__price-wraper">
+        <p class="card__price-text">Price:</p>
+        <span class="card__price-amount">$2,199</span>
+      </div>
+      <a
```
Comment (And678) [inline src/index.html:33]: See [checklist №3](https://github.com/mate-academy/layout_product-cards/blob/master/checklist.md), you need to add empty lines here.
```suggestion
      </div>
      
      <div class="card__price-wraper">
        <p class="card__price-text">Price:</p>
        <span class="card__price-amount">$2,199</span>
      </div>
      
      <a
```

**Review by And678 — CHANGES_REQUESTED** [review]
Looks good, but some things need to be improved in code-style

**Review by juljaVysotska — APPROVED** [review]
Good job!

**Pull request closed without merging**